tests/transforms/test_batch_transform.py

```diff
@@ -479,8 +481,7 @@ def circuit(x):
 
         circuit(x)
 
-        captured = capsys.readouterr()
-        assert captured.out == "custom wrapper called\n"
+        assert custom_wrapper_called[0] is True

```

Comment:
By editing a non-local variable, we don't have to capture readout.  This was causing problems for me locally.